docs-ui/stories/components/alert.stories.js

```diff
@@ -39,19 +38,19 @@ Default.parameters = {
 
 export const WithIcons = () => (
   <Grid>
-    <Alert type="info" icon={<IconInfo size="md" />}>
+    <Alert type="info" showIcon>
       <ExternalLink href="#">Info message with a url</ExternalLink>
     </Alert>
 
-    <Alert type="success" icon={<IconCheckmark size="md" />}>
+    <Alert type="success" showIcon>
       Success message without a url
     </Alert>
 
-    <Alert type="warning" icon={<IconWarning size="md" />}>
+    <Alert type="warning" showIcon>
       Warning message
     </Alert>
 
-    <Alert type="error" icon={<IconNot size="md" />}>
+    <Alert type="error" showIcon>
```

Comment:
@vuluongj20 show showIcon just be default true? seems like we add it everywhere (or least in a lot of places)